modules/microshift-fips-rpm-system.adoc

```diff
@@ -8,7 +8,7 @@
 
 Using FIPS with {microshift-short} requires enabling the cryptographic module self-checks in your {op-system-base-full} installation. After the host operating system has been configured to start with the FIPS modules, {microshift-short} containers are automatically enabled to run in FIPS mode.
 
-* When {op-system-base} is started in FIPS mode, {microshift-short} core components use the {op-system} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64 architectures.
+* When {op-system-base} is started in FIPS mode, {microshift-short} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64 architectures.
```

Comment:
Does FIPS mode only work on x86_64, and not on ARM ?

Reply:
Just updating the attribute here, not reworking any content.